packages/storybook/stories/va-banner.stories.jsx

```diff
@@ -21,14 +21,68 @@ export default {
   },
 };
 
-const Template = ({
+const DuringTemplate = ({
   'disable-analytics': disableAnalytics,
   'show-close': showClose,
   headline,
   type,
   visible,
   'window-session': windowSession,
 }) => {
+  const timeFormattingOptions = {
```

Comment:
I didn't spend much time on making this code re-usable since the banner component and its stories will [get deleted soon-ish](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/816#issuecomment-1208405693).

> Delete va-banner web component